pkg/validation-webhook/webhook.go

```diff
@@ -215,7 +230,30 @@ func main(cmd *cobra.Command, args []string) {
 		GetCertificate: cw.GetCertificate,
 	}
 
-	if err := startServer(ctx, tlsConfig, cw); err != nil {
+	// Create an indexer.
+	// Create the client config. Use kubeconfig if given, otherwise assume in-cluster.
+	config, err := buildConfig(kubeconfigFile)
+	if err != nil {
+		klog.Error(err.Error())
+		os.Exit(1)
+	}
+	snapClient, err := clientset.NewForConfig(config)
+	if err != nil {
+		klog.Errorf("Error building snapshot clientset: %s", err.Error())
+		os.Exit(1)
+	}
+
+	factory := informers.NewSharedInformerFactory(snapClient, 0)
+	lister := factory.Snapshot().V1().VolumeSnapshotClasses().Lister()
```

Comment:
Did you test it in a real cluster? I *think* it was necessary to run the informer (`go factory.Run()`) and also to wait for the informer to get populated by initial storage class list (`cache.WaitForCacheSync()`) and only then start the server.

Reply:
Why it is still a WIP :) I am working on this part now, but thanks for the pointer!